static/src/javascripts/projects/journalism/modules/render-campaign.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import fastdom from 'lib/fastdom-promise';
+import template from 'lodash/utilities/template';
+import campaignForm from 'raw-loader!journalism/views/campaignForm.html';
+import { getCampaign } from 'journalism/modules/get-campaign';
+import { submitForm } from 'journalism/modules/submit-form';
+
+const renderCampaign = (anchorNode: HTMLElement): void => {
+    const campaign = template(campaignForm, { data: getCampaign() });
+    const campaignDiv = `<figure class="element element-campaign">${
+        campaign
+    }</figure>`;
+
+    fastdom
+        .write(() => {
+            anchorNode.insertAdjacentHTML('afterend', campaignDiv);
+        })
+        .then(() => {
+            const cForm = document.querySelector('.campaign form');
+
+            if (cForm) {
+                cForm.addEventListener('submit', e => submitForm(e));
+            }
+        });
+};
+
+export const initCampaign = () => {
+    const fourthParagraph = document.querySelectorAll(
+        '.content__article-body p'
+    )[4];
```

Comment:
I _guess_ this is equivalent to `.content__article-body p:nth-of-type(4)`. I'd be interested to know if it works!

Reply:
yes good point, that would clearer